commands/zonesector.js

```diff
@@ -52,7 +42,7 @@ module.exports = {
 		const title = argv._.length ? trimString(argv._.join(' '), 100) : '';
 		const lang = await ctx.bot.getValidLanguageCode(argv.lang, ctx);
 		const fileName = `./gamedata/myz/myz-zonesectors.${lang}.yml`;
-		const h = () => argv.hide ? '||' : '';
```

Comment:
Why in the hell I did like that??

Reply:
That's what i was wondering 😛. Maybe you wanted to try this syntax out?